pkg/common/const.go

```diff
@@ -19,8 +19,8 @@ package common
 const (
 	// busyboxImage is the image used to extract the e2e logs
 	busyboxImage = "registry.k8s.io/e2e-test-images/busybox:1.36.1-1"
-	// Namespace is the namespace where the conformance pod is created
-	Namespace = "conformance"
+	// namespace is the namespace where the conformance pod is created
```

Comment:
nit:  would prefer it to be named 'Namespace' since we are utilizing 'namespace' as a variable within the code as well, just to maintain differentiation as a constant value.

Reply:
yes, that would be good